api/zulip/__init__.py

```diff
@@ -373,20 +373,20 @@ def end_error_retry(succeeded):
                 else:
                     client_cert = self.client_cert
 
-                res = requests.request(
-                    method,
-                    urllib.parse.urljoin(self.base_url, url),
-                    auth=requests.auth.HTTPBasicAuth(self.email,
-                                                     self.api_key),
-                    verify=self.tls_verification,
-                    cert=client_cert,
-                    timeout=90,
-                    headers={"User-agent": self.get_user_agent()},
-                    **kwargs)
+                s = requests.Session()
+                req = requests.Request(method,urllib.parse.urljoin(self.base_url,url),params=**kwargs)
```

Comment:
@Shashankagarwal181996 from [Travis errs](https://travis-ci.org/zulip/zulip/jobs/231580478): `params=**kwargs` should be `**kwargs`, and 4 missing whitespaces after each comma.